test/cypress/integration/card/billing-address-collection.cy.test.js

```diff
@@ -130,12 +133,9 @@ describe('Billing address collection', () => {
     const createPaymentChargeStubs = cardPaymentStubs.buildCreatePaymentChargeStubs(
       tokenId, chargeId, 'en', gatewayAccountId, serviceOpts, {}, {}, chargeOpts)
 
-    beforeEach(() => {
-      // preserve cookies so we can proceed with payment flow
-      Cypress.Cookies.preserveOnce('frontend_state')
-    })
+    it('Should not show the billing address section and allow valid card submission', () => {
+      cy.log('should not show the billing address section')
```

Comment:
It would be better if logs are close to lines relevant (line 142 in this case). setupStubs and visiting page can be on their own as they are applicable for the entire test